package chat;

message Send {
	required int32 message_id = 1;
	required string to_user = 2;
	required string content = 3;
}

message Receive {
	required int32 message_id = 1;
	required string from_user = 2;
	required string content = 3;
}

message Response {
    required int32 message_id = 1;
    enum Error {
        SUCCESS = 0;
        OTHER_OFFLINE = 1;
        WAIT_FOR_OTHER_REPLY = 2;
    }
    required Error err = 2;
}
